custom_components/ha_washdata/frontend.py

```
@@ -115,42 +149,59 @@ def __init__(self, hass: HomeAssistant) -> None:
                         "Lovelace component loaded; retrying resource registration"
                     )
+                    if unsubscribe_on_lovelace_loaded:
+                        unsubscribe_on_lovelace_loaded()
                     try:
-                        await _init_resource(self.hass, INTEGRATION_URL, _VERSION)
+                        if await _init_resource(self.hass, INTEGRATION_URL, version):
+                            self.hass.data["ha_washdata_card_registered"] = True
+                            self.hass.data["ha_washdata_card_deferred"] = False
+                        else:
+                            self.hass.data["ha_washdata_card_deferred"] = False
                     except Exception:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)